radixdlt-regression/build.gradle

```diff
@@ -121,7 +121,7 @@ dependencies {
     api project(':radixdlt-java')
     api project(':radixdlt')
 
-    api 'com.github.bitcoinj:bitcoinj:7c31dcb'
+    api 'com.github.bitcoinj:bitcoinj:0.16.1'
```

Comment:
I did this change because I was getting:
```
Caused by: org.gradle.internal.resolve.ModuleVersionNotFoundException: Could not find com.github.bitcoinj:bitcoinj:7c31dcb.
Required by:
    project :radixdlt-regression
```